### PR TITLE
Fix parsing of payload for configReportRsp

### PR DESCRIPTION
**Pull request by carldebilly**
The parsing of the payload for `configReportRsp` was incorrect and leading to `raw` packet type in `zigbee2mqtt` as defined in this comment: https://github.com/Koenkk/zigbee2mqtt/issues/2058#issuecomment-568941285

✔️ Also include mfgr clusters for Stelpro

### CODE REVIEW

**Comment by carldebilly** [comment]
Reference payload for the new test:
image

**Comment by carldebilly** [comment]
@Koenkk I suspect the failing test is incorrect.  Also I'm wordering if the parsing should be `"flat"` instead of `"repetitive"`...

**Comment by Koenkk** [comment]
@carldebilly thanks for the PR, unfortunately I'm on holiday now, will dive into it after.

**Comment by Koenkk** [comment]
The reason why this was here is: 

image

However I'm not sure what it exactly means, need to check with some devices.

**Comment by carldebilly** [comment]
You're right: it's breaking somewhere else.
I think the right fix is to switch the command parsing strategy to `flat`.

**Comment by carldebilly** [comment]
@Koenkk Ok I fixed the problem.

Now the payload is "optional" but when present, it won't crash the parsing as before.

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -1716,6 +1716,8 @@ const Cluster: {
             acCapacityFormat: {ID: 71, type: DataType.enum8},
             SinopeOccupancy: {ID: 1024, type: DataType.enum8, manufacturerCode: ManufacturerCode.Sinope},
             SinopeBacklight: {ID: 1026, type: DataType.enum8, manufacturerCode: ManufacturerCode.Sinope},
+            StelproSystemMode: {ID: 0x401c, type: DataType.enum8},
```
Comment (Koenkk) [inline src/zcl/definition/cluster.ts]: Can we also add `manufacturerCode: ManufacturerCode.Stelpro` here?

Comment (carldebilly) [reply src/zcl/definition/cluster.ts]: From what I saw it's not using the manufacturer code there...  but maybe I should check again to be sure.

Comment (carldebilly) [reply src/zcl/definition/cluster.ts]: Anyway this mode is not well supported by the converter right now. The thermostat is supporting an _eco_ mode which is not implemented.

**Comment by Koenkk** [comment]
The parsing should definitely repetitive, otherwise when multiple status records are returned only this first one will be parsed. I've updated the PR to do this, test are OK now.

@carldebilly let me know if this is ok for you then this can be merged.

**Comment by carldebilly** [comment]
I'm confortable with those changes.

**Comment by Koenkk** [comment]
Awesome, thanks!

**Pull request merged**